src/App.tsx

```diff
@@ -28,15 +38,88 @@ const MapListScreen = () => {
   return <MapList></MapList>;
 };
 
+//Created dummy About Us screen to test all three tabs
+const AboutUsScreen = () => {
+  return (
+    <View
+      style={{
+        flex: 1,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#ffffff",
+      }}
+    >
+      <Text style={{ fontSize: 30 }}>About Us</Text>
+    </View>
+  );
+};
+
+// If the focused route is not found, we need to assume it's the initial screen
+// This can happen during if there hasn't been any navigation inside the screen
+// In our case, it's "Home" as that's the first screen inside the navigator
+const getHeaderTitle = (route) => {
+  const routeName = getFocusedRouteNameFromRoute(route) ?? "Home";
+
+  switch (routeName) {
+    case "Home":
+      return "Home";
+    case "Map":
+      return "MapList";
+    case "About Us":
+      return "About Us";
+  }
+};
+
+const Tabs = ({ navigation, route }) => {
+  React.useLayoutEffect(() => {
+    navigation.setOptions({ headerTitle: getHeaderTitle(route) });
+  }, [navigation, route]);
+
+  return (
+    <Tab.Navigator
+      screenOptions={({ route }) => ({
+        headerShown: false,
+        tabBarStyle: { backgroundColor: "black" },
+        tabBarIcon: ({ color, size }) => {
+          let iconName;
+
+          if (route.name === "Home") {
+            iconName = "home";
+          } else if (route.name === "Map") {
+            iconName = "map";
+          } else if (route.name === "About Us") {
+            iconName = "information-circle-outline";
+          }
+
+          // You can return any component that you like here!
+          return <Ionicons name={iconName} size={size} color={color} />;
```

Comment:
Fix the type error for the name prop -> should be "name:string={iconName}

Reply:
Changing with this "name:string={iconName}" is giving an error "Namespace tags are not supported by default. React's JSX doesn't support namespace tags. You can set `throwIfNamespace: false` to bypass this warning". Then I tried to change with "nameString={iconName}" as per one of the solutions. It worked but icons were no rendering. So, I changed code to   
          `if (route.name === "Home") {
            return <Ionicons name='home' size={size} color={color} />;
          } else if (route.name === "Map") {
            return <Ionicons name='map' size={size} color={color} />;
          } else if (route.name === "About Us") {
            return <Ionicons name='information-circle-outline' size={size} color={color} />;
          }  `

and now working fine and rid away from type error.